content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -40,15 +40,16 @@
                                 jcr:primaryType="nt:unstructured"
                                 sling:resourceType="granite/ui/components/coral/foundation/heading"
                                 level="{Long}4"
-                                text="Client-Side Libraries"/>
+                                text="Client-Side Libraries for Body element"/>
```

Comment:
This text is not 100% correct. CSS having that category is still loaded in the head (https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/blob/8912754d5dd0a90c99b0191268dfe87878a4145c/content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/headlibs.html#L28).

Reply:
+1 we should make it clear that this is JS related, also in the java docs